src/main/java/emissary/core/IBaseDataObject.java

```diff
@@ -66,6 +67,18 @@ enum MergePolicy {
      */
     void setChannelFactory(final SeekableByteChannelFactory sbcf);
 
+    /**
+     * Returns a new InputStream to the data that this BaseDataObject contains.
+     * 
```

Comment:
Use `<p>` or `<ul>` + `<li>` markup to preserve the line breaks in the rendered Javadoc